ratis-examples/src/main/java/org/apache/ratis/examples/filestore/cli/Client.java

```diff
@@ -121,27 +133,61 @@ protected void stop(RaftClient client) throws IOException {
     System.exit(0);
   }
 
-  protected List<String> generateFiles() throws IOException {
+  public String getPath(String fileName) {
+    int hash = fileName.hashCode() % storageDir.size();
+    return new File(storageDir.get(Math.abs(hash)), fileName).getAbsolutePath();
+  }
+
+  protected void dropCache() throws InterruptedException, IOException {
+    String[] cmds = {"/bin/sh","-c","echo 3 > /proc/sys/vm/drop_caches"};
+    Process pro = Runtime.getRuntime().exec(cmds);
+    pro.waitFor();
```

Comment:
Let's try-catch it since the command is platform dependent.
```
    try {
      Process pro = Runtime.getRuntime().exec(cmds);
      pro.waitFor();
    } catch (Throwable t) {
      LOG.warn("Failed to run command " + Arrays.toString(cmds), t);
    }
```